src/smartpeak/source/ui/Widget.cpp

```diff
@@ -28,10 +28,134 @@
 #include <utility>
 #include <vector>
 #include <imgui.h>
+#include <imgui_internal.h>
 #include <implot.h>
 
 namespace SmartPeak
 {
+  struct ImTableEntry
+  {
+    int ID;
+    std::vector<const char*> Headers;
+    
+    static const ImGuiTableSortSpecs* s_current_sort_specs;
+    
+    static bool is_digit(const char ch)
```

Comment:
Are there any built-in utilities that can be used for this functionality?  I just want to double check that we are not re-inventing the wheel.  If not, then it would be great if we can add some unit testing for these methods.

Reply:
None of the implemented functions in `ImTableEntry` are available in the built-in utilities.